lib_camera/api/camera_utils.h

```diff
@@ -108,5 +109,21 @@ void io_write_bmp_file(
     const size_t width, 
     const size_t channels);
 
+/**
+ * @brief VPU-based memcpy implementation.
+ *
+ * Same as standard `memcpy()` except for an extra constraint that both `dst and `src` must be
+ * word-aligned addresses.
+ *
+ * @param[out]  dst   Destination address
+ * @param[in]   src   Source address
+ * @param[in]   bytes Number of bytes to copy
+ *
+ * @ingroup util_macros
+ */
+void vpu_memcpy(
```

Comment:
If we want to keep using it, it should have a different name as I'm planning to rename the one in lib_xcore_math as well. Could also remove from the public api, don't feel strongly about it tho

Reply:
- opt_memcpy
- xmemcpy
- cpymem
- fast_memcpy
- funny_memcpy
- aligned_memcpy

Reply:
the winner is: xmemcpy